sources/web/datalab/polymer/modules/github-cache/github-cache.html

```diff
@@ -0,0 +1,15 @@
+<!--
```

Comment:
This sounds to me like a Github utility, why not put it inside the github-file-manager element as a script file?

Reply:
Done. I moved the code to github-file-manager.ts.